css/common.css

```diff
@@ -90,7 +90,7 @@ nav {
 }
 
 
-@media only screen and (max-width: 900px) {
+@media only screen and (max-width: 1000px) {
```

Comment:
There is still some margin of 25-50 pixels. For how long are we planning to keep *Open Day* on the menu?